docs/tutorials/07_examples_vehicle_routing.ipynb

```diff
@@ -168,10 +168,7 @@
    "source": [
     "## Initialization\n",
     "\n",
-    "First of all we load all the packages that we need: \n",
-    "  - Python 3.6 or greater is required;\n",
-    "  - CPLEX 12.8 or greater is required for the classical computations;\n",
-    "  - Latest Qiskit is required for the quantum computations."
+    "First of all we load all the packages that we need. "
```

Comment:
The removed bit about CPLEX stated `is required for the classical computations`. I see the code below prints a warning and with no comment maybe it might confuse a user if we don't at least say here that, of all the packages we need, CPLEX is Optional in this tutorial and the classical computation will be skipped if its not installed. Or something like that.

I will also note that this sentence is pretty much the same as the comment that comes right after it in the code block. Maybe we can remove the comment from the code block rather than state more or less the same thing one right after the other,

Reply:
Thanks. I reverted "CPLEX is required for the classical computations" and removed the same comments in the code block.